scrapers/xSinsVR.yml

```diff
@@ -0,0 +1,37 @@
+name: "xSinsVR"
+sceneByURL: &byUrl
+  - action: scrapeXPath
+    url:
+      - xsinsvr.com/video
+      - billiestar.com/video
+    scraper: sceneScraper
+galleryByURL: *byUrl
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: &title //div[contains(@class,"video-player-container__info")]/div/h1
+      Date: &date
+        selector: //time/text()
+        postProcess:
+          - parseDate: Jan 02, 2006
+      Details: &details
+        selector: //li/div[@class="small"]/p/text()
+        concat: " "
```

Comment:
The text can be inside the p or a span in the p or an anchor among others
eg
`https://xsinsvr.com/video/leveling-up`
`https://billiestar.com/video/double-anal-2`

Using either 
```yaml
        selector: //li/div[@class="small"]
```
or 
```yaml
        selector: //li/div[@class="small"]/p//text()
        concat: " "
```
should work

Reply:
Found even better solution

```yaml
        selector: //li/div[@class="small"]//p
        concat: "\n\n"
```

This will properly separate the paragraphs and include all the texts in the <p> tags and their children.
Works on your example links.